ui/StatusQ/src/StatusQ/Controls/StatusChatInfoButton.qml

```diff
@@ -79,28 +98,29 @@ Button {
                 spacing: 1
 
                 StatusIcon {
-                    visible: root.type !== StatusChatInfoButton.Type.OneToOneChat && !forceHideTypeIcon
+                    visible: root.type !== StatusChatInfoButton.Type.OneToOneChat && !forceHideTypeIcon && icon
                     Layout.preferredWidth: 14
                     Layout.preferredHeight: 14
                     color: root.muted ? Theme.palette.baseColor1 : Theme.palette.directColor1
                     icon: {
                         switch (root.type) {
-                        case StatusChatInfoButton.Type.PublicCat:
```

Comment:
:D

Reply:
Yeah... evil typo 😸

The worst thing is that it didn't produce any warning whatsoever